webapp/app.py

```diff
@@ -96,17 +96,11 @@ def department_group(department):
     )
 
 
-@app.route("/careers/<department>/<job_id>")
-def job_details(department, job_id):
+@app.route("/careers/job/<job_id>")
```

Comment:
is that a url that we want? `/careers/job` sounds a bit redoundant no?

Reply:
the job id might be something unique. (`/careers/job/1` for example) so a url that we will never use. Why not just putting `/careers/1` feels shorter and cleaner URL.

Reply:
even what would be great would be to have the slug instead of the number but I don't know if greenhouse provides this.

Reply:
@tbille greenhouse only takes id to identify the job. To be honest I'm not against using `/careers/1` instead of `/careers/job/1`, but I thought `/careers/job/1` it makes more sense as it makes it clear you are looking at a job.

Reply:
>  greenhouse only takes id to identify the job. 

ok

> To be honest I'm not against using /careers/1 instead of /careers/job/1, but I thought /careers/job/1 it makes more sense as it makes it clear you are looking at a job.

Well when you are in the `/careers/*` it already implies that you are looking for a job.

Reply:
@tbille will change it to `/careers/1` as per your suggestion. thanks